test/bedrock_Azure_mc_test.go

```diff
@@ -0,0 +1,234 @@
+package test
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+	"time"
+	"io/ioutil"
+	"log"
+
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	"github.com/gruntwork-io/terratest/modules/random"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/gruntwork-io/terratest/modules/http-helper"
+)
+
+func addIPtoYAML(input string, ipaddress string) {
```

Comment:
Are you adding the annotation for the static ip to work on non node resource group?